src/write.c

```diff
@@ -1333,8 +1351,9 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
     }
 
     avifBoxMarker ftyp = avifRWStreamWriteBox(&s, "ftyp", AVIF_BOX_SIZE_TBD);
-    avifRWStreamWriteChars(&s, majorBrand, 4);                             // unsigned int(32) major_brand;
-    avifRWStreamWriteU32(&s, 0);                                           // unsigned int(32) minor_version;
+    avifRWStreamWriteChars(&s, majorBrand, 4); // unsigned int(32) major_brand;
+    // TODO(yguyon): AV2-AVIF is AVIFv2 for now (change once no longer experimental)
+    avifRWStreamWriteU32(&s, isAV2 ? 2 : 0);                               // unsigned int(32) minor_version;
```

Comment:
does it say anywhere that minor_version == 2 means it's experimental?

just curious as to why the value is 2.

Reply:
> does it say anywhere that minor_version == 2 means it's experimental?

No.

> just curious as to why the value is 2.

Because major brand `avif` minor version 0 uses AV1. `avif` version 1 is confusing, so I used `avif` version 2 for AV2. Maybe it will be another brand such as `av2f` version 0 anyway.

Clarified the comment.